### ISSUE TITLE
Screen brightness do not set to zero ...

### ISSUE THREAD

**Issue by Target0815** [FrSkyRC/ETHOS-Feedback-Community#1986]
Hi,

... otherwise it will be difficult to get the display readable again ;).

The minimum setting should still be readable, but not black.

**Comment by Wolepo** [comment]
Correct, this is how it should be with all types of transmitters.

**Comment by Beastyboy80** [comment]
Good idea

**Comment by NorbertHeli** [comment]
Also with lamp you can´t see anything when display set to zero

**Comment by Target0815** [comment]
Exactly, which is why the darkening should only be possible to the extent that you can still just make out something.

**Comment by Beastyboy80** [comment]
Which Transmitter ?

**Comment by bsongis-frsky** [comment]
@Target0815 Which radio. I have just tested on X20S, it behaves exactly the way you describe

**Comment by Target0815** [comment]
I have an X18S. But that doesn't matter. It should not be possible to set the brightness to zero in the configuration, because newcomers would then sit in front of a black screen and not be able to activate it again.

**Comment by bsongis-frsky** [comment]
It is better if you give this information as soon as possible. Here it would already have been fixed.

**Comment by bsongis-frsky** [comment]
Two tests made by different people this morning, X18 backlight set at minimum is perfectly readable.

Someone else?

**Comment by Target0815** [comment]
I'm still trying to get info on transmitter and ethos version from whoever reported this on the local forum ....

**Comment by Target0815** [comment]
The user has since updated his transmitter and no longer has a problem.

I am therefore closing the issue.

**Comment by MIKE-DAILEY** [comment]
It would be very helpful if in the future, that issues are checked carefully before posting.  Having the development team pulled away from development work slows down progress on ETHOS
